src/python/grpcio_tests/tests/unit/_invocation_defects_test.py

```diff
@@ -199,6 +177,7 @@ def _defective_handler_multi_callable(channel):
 
 
 class InvocationDefectsTest(unittest.TestCase):
+    """Tests the handling of exception-raising user code."""
```

Comment:
Can we explicitly add "on the client side" to this comment?

Reply:
Done.